python/ml4ir/applications/ranking/tests/test_lr_schedule.py

```diff
@@ -27,6 +27,7 @@
 
 INPUT_DIR = "ml4ir/applications/ranking/tests/data/"
 MODEL_CONFIG = "ml4ir/applications/ranking/tests/data/configs/model_config_cyclic_lr.yaml"
+MODEL_CONFIG_REDUCE_LR_ON_PLATEAU = "ml4ir/applications/ranking/tests/data/configs/model_config_reduce_lr_on_plateau.yaml"
```

Comment:
What do you think about not committing actual files in git (like the config you committed) and simply writing/removing then in setUp, tearDown ? The advantage being that you have them in front of your eyes when the debug the test.

Reply:
I see your point, but since this is the way ml4ir is operating (reading config files) I followed it.